joomla/joomla-cms
# [4.4] com_tags extra parent_id query var

### Summary of Changes

`parent_id` query var is not stripped for the same view=tags menu item 

### Testing Instructions

1. Create a menu item of 'Tags > List All Tags' type, select 'Root' in 'Parent_tag'.
2. Ensure that you have any tags and tagged articles.
3. Enable 'Show Tags' in any article menu item, click tag to access tag-specific page (`option=com_tags&view=tag`)
4. Ensure that you have breadcrumbs displayed, see the menu item from p.1 with extra `parent_id` query params like https://site.com/tags?parent_id=1

### Actual result BEFORE applying this Pull Request

Useless `parent_id` query var exists in the URL.

### Expected result AFTER applying this Pull Request

`parent_id` query var is stripped if the menu item has the same `parent_id`

### Link to documentations
Please select:
